tensorflow-jupyter/tensorflow-deployment.yaml

```diff
@@ -60,14 +60,14 @@ spec:
       # Read more about affinity at: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
       affinity:
         nodeAffinity:
-          # This will REQUIRE the Pod to be run on a system with a Pascal GPU
+          # This will REQUIRE the Pod to be run on a system with an NVIDIA A40 GPU
           requiredDuringSchedulingIgnoredDuringExecution:
             nodeSelectorTerms:
             - matchExpressions:
               - key: gpu.nvidia.com/class
                 operator: In
                 values:
-                  - NV_Pascal
+                  - A40
               - key: failure-domain.beta.kubernetes.io/region
```

Comment:
We should also update keys like these to be up-to-date when we can (e.g. `failure-domain.beta.kubernetes.io/region` → `topology.kubernetes.io/region`)